<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestItemInterface</class>
 <widget class="QWidget" name="TestItemInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>69</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="lbIndex">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbComName">
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>底板串口</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>产品串口</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbEncode">
        <property name="minimumSize">
         <size>
          <width>57</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>HEX</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>H2S</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leTx">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>发送</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spbxDelay">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>86</width>
          <height>0</height>
         </size>
        </property>
        <property name="suffix">
         <string>s</string>
        </property>
        <property name="prefix">
         <string>延时</string>
        </property>
        <property name="maximum">
         <number>99</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>显示</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbTestType">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>AT</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AT&lt;\r\n&gt;</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AT&lt;HEX&gt;</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>68</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>串口查询真</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>串口查询假</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>单按钮弹框</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>双按钮弹框</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leRx">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
        <property name="placeholderText">
         <string>接收</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spbxTimeout">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="suffix">
         <string>s</string>
        </property>
        <property name="prefix">
         <string>超时</string>
        </property>
        <property name="maximum">
         <number>99</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
